WFLY-11408: Upgrade HAL to 3.1.1.Final

hpehl opened the pull request:
https://issues.jboss.org/browse/WFLY-11408

jamezp committed:
[WFLY-10995] Upgrade Netty from 4.1.25.Final to 4.1.29.Final.

There was also a required change to explicitly set the InternalLoggerFactory to avoid a class loading warning logged.

ehsavoie committed:
[WFLY-10493]: Support JMS Java EE resources definition for remote Artemis-based broker.

* Creating queue / topic on deployments
* Deleting queue / topic in undeployments
* Looking for default ee resource adapter if none is specified.
* Making @JMSConnectionFactoryDefinition work without internal broker.
* Adding support for description of runtime external destinations.
* Adding support for management address definition in the @JMSDestinationDefinition.
* Fixing WFWIP-147: @JMSConnectionFactoryDefinition with property resourceAdapater to pooled-connection-factory defined in messaging-active subsystem does not work.
* Adding support for authentication of management queue.

JIRA: https://issues.jboss.org/browse/WFLY-10493

ehsavoie committed:
[WFLY-10326]: Add setting to control global memory usage for messaging servers.

* Adding global-max-disk-usage to define the maximum percentage of disk space to use before stopping the PAGING.
* Adding disk-scan-period to define the pfrequency at which disk space usage is computed.
* Adding global-max-memory-size to define the maximal amount of memory to use for the messaging server.

JIRA: https://issues.jboss.org/browse/WFLY-10326

jmesnil committed:
[WFLY-11529] Expose WildFly metrics in Prometheus HTTP endpoint

* add WildFly metrics (from subsystems and deployments) to the /metrics
  HTTP endpoint (only in the Prometheus text format)
* Use io.prometheus simpleclient 0.6.0 to collect the metrics and write
  them in TextFormat on the /metrics HTTP endpoint
* Name and labels of metrics are built based on the resource path
  address and the attribute name (plus an optional prefix that is set
  to `wildfly` in the standalone profiles).
* Update Admin guide

JIRA: https://issues.jboss.org/browse/WFLY-11529

ehsavoie committed:
[WFLY-10493]: Support JMS Java EE resources definition for remote Artemis-based broker.

* Simplifying code and reducing coupling with Artemis.
* Fixing WFWIP-149: creating external queues in a remote broker cluster.
Jira: https://issues.jboss.org/browse/WFLY-10493
      https://issues.jboss.org/browse/WFWIP-149

hpehl committed:
WFLY-11408: Upgrade HAL to 3.1.1.Final

ropalka committed:
[WFLY-11441] Eliminating dead code.

scottmarlow committed:
WFLY-11436 also remove NON_XA from WildFly high availability configuration (also update galleon-pack/src/main/resources/feature_groups/infinispan-dist.xml + galleon-pack/src/main/resources/feature_groups/infinispan-local.xml)

jamezp committed:
Explicitly exclude Netty dependencies

kabir committed:
Merge pull request #11904 from yersan/bugs/WFLY-11452

[WFLY-11452] Clean up jboss-modules dependencies from Weld beanvalidation, transaction and webservices

kabir committed:
Merge pull request #11890 from jamezp/WFLY-11340

[WFLY-11340] Fix MP OpenTracing tests to work with the security manag…

yersan committed:
[WFLY-11460] Validate requirement for modules previously exported by javax.ejb.api on org.jboss.as.webservices

kabir committed:
Merge pull request #11885 from ochaloup/WFLY-7304-compensations-to-XTS

[WFLY-11421] compensation code is dependent on XTS code - move activates to xts subsystem

kabir committed:
Merge pull request #11849 from tadamski/WFLY-9896

[WFLY-9896] Add rule for jta authority authentication

jmesnil committed:
[WFLY-11471] Add expressions to enable statistics

* Add expressions to the features that provides statistics to enable
them using System properties
  * wildfly.statistics-enabled - Enable all statistics
  * wildfly.undertow.statistics-enabled - Enable Undertow statistics
  * wildfly.transactions.statistics-enabled - Enable Transactions
    statistics
  * wildfly.messaging-activemq.statistics-enabled - Enable Artemis
    statistics
  * wildfly.datasources.statistics-enabled - Enable Datasource
    statistics

The specific system properties (e.g.
wildfly.undertow.statistics-enabled) have precedence over the general
one (wildfly.statistics-enabled).

By default, these expressions resolves to false.

These expressions only applies to resources defined in the shipped
profiles. If a new datasource resource is created, its
`statistics-enabled` attribute will not be set to this expression.

* Update legacy subsystem-templates (to keep them in sync)

JIRA: https://issues.jboss.org/browse/WFLY-11471

yersan committed:
[WFLY-11467] Validate requirement for modules previously exported by javax.ejb.api on org.jboss.iiop-client

kabir committed:
Merge pull request #11909 from yersan/subtask-5966/WFLY-11455

[WFLY-11455] Validate requirement for modules previously exported by javax.ejb.api on org.jboss.narayana.txframework

kabir committed:
Merge pull request #11908 from yersan/subtask-5966/WFLY-11454

[WFLY-11454] Validate requirement for modules previously exported by javax.ejb.api on org.jboss.metadata.appclient

darranl committed:
[WFLY-7186] Add documentation to activate EE security with WildFly Elytron.

pferraro committed:
WFLY-11469 Workaround for ISPN-9761.

kabir committed:
Merge pull request #11891 from pferraro/WFLY-11254

WFLY-11254 Optimize FineSessionAttributes cache operations

tomjenkinson committed:
WFLY-11473 Upgrade Narayana to 5.9.1.Final

TomasHofman committed:
[WFLY-11487]: Bump messaging schema version to 6.0

ehsavoie committed:
[WFLY-11474]: Make activemq dependency on cli module optional.

* Updating the module configuration

Jira: https://issues.jboss.org/browse/WFLY-11474

ehsavoie committed:
[WFLY-11478]: Add test to cover a messaging subsystem without any internal broker.

* Adding a test to cover a deployment without an internal broker.

Jira: https://issues.jboss.org/browse/WFLY-11478

kabir committed:
Merge pull request #11915 from pferraro/infinispan

WFLY-11469 Workaround for ISPN-9761.

pferraro committed:
Remove obsolete method override.

kabir committed:
Merge pull request #11923 from pferraro/master

Prune obsolete method override.

kabir committed:
Merge pull request #11919 from ehsavoie/WFLY-11474

[WFLY-11474]: Make activemq dependency on cli module optional.

kabir committed:
Merge pull request #11917 from jbosstm/WFLY-11473

WFLY-11473 Upgrade Narayana to 5.9.1.Final

kabir committed:
Merge pull request #11912 from ropalka/WFLY-11441

[WFLY-11441] Eliminating dead code.

kabir committed:
Merge pull request #11911 from baranowb/WFLY-10808_v2

[WFLY-10808] - Default Value of local-receiver-pass-by-value in jboss…

kabir committed:
Merge pull request #11910 from yersan/subtask-5966/WFLY-11460

[WFLY-11460] Validate requirement for modules previously exported by javax.ejb.api on org.jboss.as.webservices

jamezp committed:
Merge pull request #11922 from ehsavoie/WFLY-11478

[WFLY-11478]: Add test to cover a messaging subsystem without any internal broker.

jamezp committed:
Merge pull request #11790 from ehsavoie/WFLY-10326

[WFLY-10326]: Add setting to control global memory usage for messaging servers.

ehsavoie committed:
[WFLY-11493]: Core queues loaded at boot are not removable.

* Creating service for core queues created via ActiveMQServer
configuration in ServerAdd.

Jira: https://issues.jboss.org/browse/WFLY-11493

pferraro committed:
WFLY-11490 Upgrade Infinispan to 9.4.4.Final

kabir committed:
Merge pull request #11925 from pferraro/infinispan

WFLY-11490 Upgrade Infinispan to 9.4.4.Final

kabir committed:
Merge pull request #11916 from jmesnil/WFLY-11471_statistics-enabled.expressions

[WFLY-11471] Add expressions to enable statistics

jmesnil committed:
[WFLY-11502] Upgrade WildFly Core 8.0.0.Beta1

* update XML documents to the xmlns="urn:jboss:domain:10.0" schema

JIRA: https://issues.jboss.org/browse/WFLY-11502

rgdoliveira committed:
WFLY-11376 ironjacamar 1.4.12.Final

rhusar committed:
Revert "[WFLY-10125] missing permission into EntityListenerBeanManagerInjectionTestCase"

This reverts commit eb486756da517ff4c5669f01a9f9c30d414bda91.

soul2zimate committed:
[WFLY-11503] move getClassLoader action into doPrivileged block.

rhusar committed:
WFLY-11509 SessionIdentifierSerializer should use privileged block for service loader

Signed-off-by: Radoslav Husar <radosoft@gmail.com>

kabir committed:
Merge pull request #11926 from ehsavoie/WFLY-11493

[WFLY-11493]: Core queues loaded at boot are not removable.

pferraro committed:
Reusable Service implementation for service installation into a child target.

pferraro committed:
Eliminate unnecessary intermediate collection.  Remove superfluous lambda.

pferraro committed:
Fix usage of deprecated API.

yersan committed:
[WFLY-11506] Add required org.jboss.weld.core to transaction related modules

pferraro committed:
WFLY-11521 Race condition in InfinispanRouteLocator logic can cause missing route

jmesnil committed:
[WFLY-11519] Add missing measurement unit to metrics

* Update undertow metrics

JIRA: https://issues.jboss.org/browse/WFLY-11519

jmesnil committed:
[WFLY-11519] Add missing measurement unit to metrics

* Update transaction  metrics

JIRA: https://issues.jboss.org/browse/WFLY-11519

pferraro committed:
Ensure ServiceSupplierDependency implements equals(...) and hashCode().

rhusar committed:
WFLY-11531 SimpleMarshallingContext#createMarshaller should run in a privileged block

kabir committed:
Merge pull request #11930 from soul2zimate/WFLY-11503-master

[WFLY-11503] move getClassLoader action into doPrivileged block.

kabir committed:
Merge pull request #11936 from pferraro/clustering

WFLY-11521 Race condition in InfinispanRouteLocator logic can cause missing route

kabir committed:
Merge pull request #11935 from yersan/bugs/WFLY-11506

[WFLY-11506] Add required org.jboss.weld.core to transaction related modules

kabir committed:
Merge pull request #11932 from rhusar/WFLY-11512

WFLY-11512 Remove test workaround for CDI-727

kabir committed:
Merge pull request #11931 from rhusar/WFLY-11509

WFLY-11509 SessionIdentifierSerializer should use privileged block fo…

kabir committed:
Merge pull request #11933 from jamezp/netty-exclusions

Explicitly exclude Netty dependencies

jamezp committed:
Merge pull request #11939 from ehsavoie/WFLY-11487

[WFLY-11487]: Bump messaging schema version to 6.0

jamezp committed:
[WFLY-11526] Remove the modular JVM arguments for mixed domain tests

pferraro committed:
Replace custom Time class with java.time.Duration.

pferraro committed:
CacheFactoryBulider SPI should not reference org.wildfly.clustering.ejb.spi classes.

stuartwdouglas committed:
WFLY-11528 Wildfly HTTP Client 1.0.13.Final

pferraro committed:
Move CacheFactory installation logic to StatefulComponentDescription so that we have access to a CapabilityServiceSupport.

ehsavoie committed:
[WFLY-11480]: Possible to remove referenced activemq http-connector.
 * Adding support for capabilities in connection factories and transport configurations.
 * Adding support for proper default configuration for external pcf.

Jira: https://issues.jboss.org/browse/WFLY-11480

nziakova committed:
WFLY-11510 Fix EarOpenTracingWithWeldProbeTestCase

pferraro committed:
Replace CacheFactoryBuilder.installDeploymentUnitDependencies(...) with getDeploymentServiceConfigurators(...)

kabir committed:
[WFLY-11532] Upgrade license plugin to 2.0.0.Beta2

kabir committed:
Merge pull request #11943 from rhusar/WFLY-11531

WFLY-11531 SimpleMarshallingContext#createMarshaller should run in a privileged block

kabir committed:
Merge pull request #11941 from stuartwdouglas/WFLY-11528

WFLY-11528 Wildfly HTTP Client 1.0.13.Final

kabir committed:
Merge pull request #11938 from jmesnil/WFLY-11519_time-based_metrics_measurement_unit

[WFLY-11519] Add missing measurement unit to metrics

kabir committed:
Merge pull request #11928 from jmesnil/WFLY-11502_upgrade_wildfly-core_8.0.0.Beta1

[WFLY-11502] Upgrade WildFly Core 8.0.0.Beta1

baranowb committed:
[WFLY-11364] fix JpaDsRestartTestCase with secmgr

pferraro committed:
WFLY-11511 Upgrade Infinispan to 9.4.5.Final

ehsavoie committed:
[WFLY-9305]: ActiveMQ Resource Adapter can not lookup destination in app namespace.

* Passing the NamespaceContextSelector around in the RA to be able to
access it when looking up for the resource.

Jira: https://issues.jboss.org/browse/WFLY-9305

pferraro committed:
WFLY-11530 Distributed @Stateful EJB implementation should not hold references to CapabilityServiceSupport

jamezp committed:
Merge pull request #11940 from jamezp/WFLY-11526

[WFLY-11526] Remove the modular JVM arguments for mixed domain tests

pferraro committed:
WFLY-11534 DistributableCacheFactoryServiceConfigurator should generate ServiceName for TSR via capability name

pferraro committed:
Use PrivilegedAction for BeanManagerFactoryServiceConfiguratorFactoryProvider loading.

baranowb committed:
[WFLY-11501] - fix DynamicJaspiTestCase with secmgr

Commit:
[WFLY-11536] host controller does not define native interface out of box
since WF14

kabir committed:
Merge pull request #11950 from pferraro/infinispan

WFLY-11511 Upgrade Infinispan to 9.4.5.Final

hpehl committed:
WFLY-11537: Upgrade HAL to 3.0.10.Final

kabir committed:
Merge pull request #11947 from kabir/WFLY-11532

[WFLY-11532] Upgrade license plugin to 2.0.0.Beta2

bstansberry committed:
[WFLY-11543] Revert use of ContextTransactionSynchronizationRegistry from WFLY-10426

bstansberry committed:
[WFLY-11543] Revert use of ContextTransactionSynchronizationRegistry from WFLY-10175

bstansberry committed:
[WFLY-11543] Remove use of ContextTransactionSynchronizationRegistry in agroal

sgilda committed:
Fix AsciiDoc files in the docs folder that are causing build errors and warnings

pferraro committed:
WFLY-11545 max-size of passivation-store should require reload when modified, as this attribute cannot be applied dynamically at runtime.

bstansberry committed:
Merge pull request #11961 from bstansberry/WFLY-11543

[WFLY-11543] Server-side services outside the transactions subsystem should not use ContextTransactionSynchronizationRegistry

kabir committed:
Merge pull request #11956 from hpehl/WFLY-11537

WFLY-11537: Upgrade HAL to 3.0.10.Final

tmiyargi committed:
[WFLY-11547] Inconsistency in JCA Subsystem xsd, boundedqueque is used for worker threads

baranowb committed:
[WFLY-11361] - fix LookupTest to work with secmgr

ehsavoie committed:
[WFLY-11552]: Fixing licence headers using Jboss by Red Hat in copyright notice.

* Replacing 'Jboss by Red Hat' by 'Red Hat, Inc.' for copyright notice.

Jira: https://issues.jboss.org/browse/WFLY-11552

bstansberry committed:
Merge pull request #11953 from baranowb/WFLY-11501

[WFLY-11501] - fix DynamicJaspiTestCase with secmgr

bstansberry committed:
Merge pull request #11968 from baranowb/WFLY-11361

[WFLY-11361] - fix LookupTest to work with secmgr

bstansberry committed:
Merge pull request #11971 from ehsavoie/WFLY-11552

[WFLY-11552]: Fixing licence headers using Jboss by Red Hat in copyright notice.

skyllarr committed:
[WFLY-11479] Add note to documentation about silent mode for HTTP BASIC auth

darranl committed:
[WFLY-7186] Add permission ElytronPermission("getSecurityDomain") to deployment as direct access to the SecurityDomain is used.

darranl committed:
[WFLY-11555] Only restore the old policy if this service did initialise the policy.

jamezp committed:
[WFLY-11556] Add a utility to skip tests if the security manager is enabled.

jamezp committed:
[WFLY-11365] Ignore the JSONBTestCase if the security manager is enabled.

jamezp committed:
[WFLY-11499] Ignore the MicroProfileMetricsApplicationTestCase if the security manager is enabled.

jamezp committed:
[WFLY-11357] Ignore the RemoteLocalCallTestCase if the security manager is enabled.

jamezp committed:
[WFLY-11347] Ignore the DiscoveryGroupExternalMessagingDeploymentTestCase if the security manager is enabled.

jamezp committed:
[WFLY-11346] Add permissions for the properties and environment variables for the MicroProfileConfigTestCase.

jamezp committed:
[WFLY-11337] Ignore the VerifyHibernate51Compatibility* if the security manager is enabled.

jamezp committed:
Merge pull request #11948 from baranowb/WFLY-11364

[WFLY-11364] fix JpaDsRestartTestCase with secmgr

jamezp committed:
Merge pull request #11951 from ehsavoie/WFLY-9305

[WFLY-9305]: ActiveMQ Resource Adapter can not lookup destination in …

jamezp committed:
Merge pull request #11967 from tmiyargi/JBEAP-15843-blocking

[WFLY-11547] Inconsistency in JCA Subsystem xsd, boundedqueque is use…

jamezp committed:
Merge pull request #11973 from darranl/WFLY-11555

[WFLY-11555] Only restore the old policy if this service did initialise the policy.

jamezp committed:
Merge pull request #11974 from jamezp/WFLY-11556

Some changes to get the test suite to pass with the security manager enabled

Commit:
[WFLY-11479] Tests

gsmet committed:
[WFLY-11560] Upgrade Hibernate Validator to 6.0.14.Final

ehsavoie committed:
[WFLY-11347]: Tests from org.jboss.as.test.integration.messaging fails with security manager.

* Fixing the failing tests except NotClosingInjectedContextTestCase.
* Creating https://issues.jboss.org/browse/WFLY-11561 to track the fact
that NotClosingInjectedContextTestCase is not fixed.

Jira: https://issues.jboss.org/browse/WFLY-11347

ehsavoie committed:
[WFLY-11561]: org.jboss.as.test.integration.messaging.jms.context.notClosingInjectedContext.NotClosingInjectedContextTestCase.

* Fixing missing permissions for NotClosingInjectedContextTestCase.

Jira: https://issues.jboss.org/browse/WFLY-11561

bstansberry committed:
Merge pull request #11976 from gsmet/WFLY-11560-master

[WFLY-11560] Upgrade Hibernate Validator to 6.0.14.Final - master branch

jamezp committed:
[WFLY-11556] Check the security.manager system property rather than the System.getSecurityManager(). The latter requires the check to be used in container while the former the check can be used in a @Before or @BeforeClass method.

jamezp committed:
[WFLY-11349] Ignore clustering tests that fail when the security manager is enabled. Once Infinispan is upgraded these can be reenabled. See WFLY-11539.

jamezp committed:
Merge pull request #11979 from jamezp/fix-sm-check

[WFLY-11556] Check the security.manager system property rather than t…

jamezp committed:
[WFLY-6184] Ignore Hibernate tests that don't work with the ServiceLoader when the SecurityManager is enabled.

jamezp committed:
[WFLY-11359] Ignore the ClassFileTransformerTestCase if the security manager is enabled.

bstansberry committed:
Merge pull request #11929 from rgdoliveira/WFLY-11376

[WFLY-11376] ironjacamar 1.4.12.Final

bstansberry committed:
Merge pull request #11945 from nziakova/wfly11510

WFLY-11510 Fix EarOpenTracingWithWeldProbeTestCase

bstansberry committed:
Merge pull request #11962 from sgilda/fix-doc-build-issues

Fix the AsciiDoc files in the docs folder that are causing build errors and warnings

bstansberry committed:
Merge pull request #11977 from ehsavoie/WFLY-11347

[WFLY-11347]: Tests from org.jboss.as.test.integration.messaging fails with security manager.

bstansberry committed:
Merge pull request #11980 from jamezp/WFLY-11349

[WFLY-11349] Ignore clustering tests that fail when the security mana…

bstansberry committed:
Merge pull request #11955 from mchoma/WFLY-11536

[WFLY-11536] enable-elytron.cli fails with WF15

bstansberry committed:
Merge pull request #11856 from rsearls/WFLY-11339-ServiceRef-TestCases

[WFLY-11339] FilePermission, read only, granted to ALL FILES because …

jamezp committed:
[WFLY-11348] Ignore the EESecurityAuthMechanismTestCase if the security manager is enabled.

jamezp committed:
[WFLY-9464] Ignore XACML tests if the security manager is enabled.

jamezp committed:
[WFLY-11346] Follow up to add the correct permissions to the MicroProfileConfigConvertersTestCase, ConfigSourceFromClassTestCase and ConfigSourceProviderFromClassTestCase tests.

scottmarlow committed:
WFLY-11563 Race condition in PersistenceUnitServiceImpl when the service is being stopped

jamezp committed:
[WFLY-11357] Ignore the RemoteLocalCallProfileTestCase if the security manager is enabled.

jamezp committed:
Merge pull request #11982 from jamezp/sm-integ-basic

Finish ignoring for fixing tests when the security manager is enabled

spyrkob committed:
[WFLY-11571] Upgrade FasterXML to 2.9.8

jbliznak committed:
[WFLY-11573] Remove unused Servlet API 3.1 dependency

jamezp committed:
[WFLY-11359] Ignore the ClassFileTransformerTestCase if the security manager is enabled.

jamezp committed:
Add permissions to read java.net.preferIPv4Stack and java.net.preferIPv6Addresses for WS trust tests.

pferraro committed:
WFLY-11575 DefaultCacheContainer getCache(...) createIfAbsent variants throw NPE when called with false and cache does not exist

spyrkob committed:
[WFLY-11576] Upgrade common-digester to 1.8.1

jamezp committed:
Merge pull request #11987 from jamezp/sm-fixes

Final (hopefully) security manager test suite fixes/ignores

jmesnil committed:
[WFLY-11577] Upgrade WildFly Core 8.0.0.Beta2

JIRA: https://issues.jboss.org/browse/WFLY-11577

jmesnil committed:
[WFLY-11578] Distinguish WildFly metrics types

* In microprofile-metrics-smallrye extension, when attributes that
  corresponds to metrics are collected, check whether the attribute
  access has the COUNTER_METRIC flag to register it in Prometheus
  collector as a counter. Otherwise, the metric is registered as a
  gauge.
* Append _total to the Prometheus name for counters.

JIRA: https://issues.jboss.org/browse/WFLY-11578

jmesnil committed:
[WFLY-11578] Distinguish WildFly metrics

* Flag EJB3 metrics that represents counters with
  COUNTER_METRIC.

JIRA: https://issues.jboss.org/browse/WFLY-11578

jmesnil committed:
[WFLY-11578] Distinguish WildFly metrics

* Flag Transactions metrics that represents counters with
  COUNTER_METRIC.

JIRA: https://issues.jboss.org/browse/WFLY-11578

jmesnil committed:
[WFLY-11578] Distinguish WildFly metrics

* Flag messaging-activemq metrics that represents counters with
  COUNTER_METRIC.

JIRA: https://issues.jboss.org/browse/WFLY-11578

kabir committed:
Merge pull request #11991 from jmesnil/WFLY-11577_upgrade_wildfly-core_8.0.0.Beta2

[WFLY-11577] Upgrade WildFly Core 8.0.0.Beta2

kabir committed:
Merge pull request #11984 from spyrkob/WFLY-11571

[WFLY-11571] Upgrade FasterXML to 2.9.8

kabir committed:
Merge pull request #11983 from scottmarlow/WFLY-11563_PersistenceUnitServiceImplNPE

WFLY-11563 Race condition in PersistenceUnitServiceImpl when the service is being stopped

kabir committed:
Merge pull request #11978 from spyrkob/WFLY-10995

[WFLY-10995] Upgrade Netty from 4.1.25.Final to 4.1.29.Final.

sgilda committed:
Set appropriate language for source blocks and set nowrap option

jmesnil committed:
[WFLY-11578] Distinguish WildFly metrics

* Flag Undertow metrics that represents counters with COUNTER_METRIC.

JIRA: https://issues.jboss.org/browse/WFLY-11578

tmiyargi committed:
[WFLY-11584] Legacy Web migrate op fails if a connector has scheme https and no SSL config

ehsavoie committed:
[WFLY-11586]: Missing runtime reference to external destinations.

* Fixing the missing reference to rcternal JMS destinations in deployments.

Jira: https://issues.jboss.org/browse/WFLY-11586

pferraro committed:
Add class table entries for non-random access collection wrapper types.

pferraro committed:
Avoid use of synthetic accessor method.

pferraro committed:
WFLY-11585 Access ordered LinkedHashMap not externalized correctly

kabir committed:
Merge pull request #11994 from pferraro/marshalling

 WFLY-11585 Access ordered LinkedHashMap not externalized correctly

ehsavoie committed:
[WFLY-11579]: Description of transaction timeout is misleading.

* Fixig description

Jira: https://issues.jboss.org/browse/WFLY-11579

Commit:
[WFLY-11594] Prepare tests for MicroProfile Metrics issues 32,33 and 34

bstansberry committed:
[WFLY-11587] Fix Weld integration with legacy security

1) Correct SecuritySubsystemRootResourceDefinition so it actually registers the org.wildfly.legacy-security.server-security-manager capability
2) Correct the module import as ServerSecurityManager is not from the security subsystem module, it's from WildFly Core's org.jboss.as.core.security.
3) Add a test of the use of @Inject Principal

pferraro committed:
Fix array overallocation.

gaol committed:
[WFLY-11352] WildFly registers multiple distinct drivers for current MySQL driver jar

kabir committed:
Merge pull request #11942 from ehsavoie/WFLY-11480

[WFLY-11480]: Possible to remove referenced activemq http-connector.

marekkopecky committed:
[WFLY-11594] Prepare tests for different Prometheus and json values in MicroProfile Metrics

jbaesner committed:
WFLY-11590 Cancelled timer not removed from internal cache when using persistent timers

yersan committed:
[WFLY-11591] Remove org.glassfish:javax.json and javax.json:javax.json-api licences from servlet-feature-pack-licenses.xml

jstourac committed:
[WFLY-11593] Typo and ambiguity in 'integrated-jaspi' attribute of 'application-security-domain' element.

kabir committed:
Merge pull request #11995 from ehsavoie/WFLY-11579

[WFLY-11579]: Description of transaction timeout is misleading.

kabir committed:
Merge pull request #11988 from pferraro/infinispan

WFLY-11575 DefaultCacheContainer getCache(...) createIfAbsent variants throw NPE...

yersan committed:
[WFLY-11592] Missing microprofile-config-smallrye subsystem in some configuration files

Commit:
[WFLY-11594] Prepare tests for MicroProfile Metrics issues 42 and 43

pferraro committed:
WFLY-11595 Eliminate unnecessary remote command invocation when EJB has no @StatefulTimeout

spyrkob committed:
[WFLY-11599] Upgrade jboss-ejb-client from 4.0.12.Final to 4.0.13.Final

chengfang committed:
WFLY-11580 Upgrade jberet-core from 1.3.2.Final to 1.3.3.Final.

pferraro committed:
WFLY-11597 Allow eager SFSB passivation when enabled via system property

spriadka commented:
Please note that following issues incorporated by the linked JIRA don't have test dev done:
 * https://issues.jboss.org/browse/HAL-1554
 * https://issues.jboss.org/browse/HAL-1523
 * https://issues.jboss.org/browse/HAL-532
 * https://issues.jboss.org/browse/HAL-1500
 * https://issues.jboss.org/browse/HAL-1504
 * https://issues.jboss.org/browse/HAL-1485
 * https://issues.jboss.org/browse/HAL-1528
 * https://issues.jboss.org/browse/HAL-1545

jmesnil committed:
[WFLY-11588] eliminates overrides of SimpleResourceDefinition.registerCapabilities

Configure capabilities using
org.jboss.as.controller.SimpleResourceDefinition.Parameters#addCapabilities
instead of overriding SimpleResourceDefinition.registerCapabilities
(unless a specific treatment is required).

When registerCapabilities is overridden, always call
super.registerCapabilities()

JIRA: https://issues.jboss.org/browse/WFCORE-4276

kabir committed:
Merge pull request #12007 from pferraro/WFLY-11595

WFLY-11595 Eliminate unnecessary remote command invocation when EJB has no @StatefulTimeout

kabir committed:
Merge pull request #12005 from chengfang/WFLY-11580

WFLY-11580 Upgrade jberet-core from 1.3.2.Final to 1.3.3.Final.

kabir committed:
Merge pull request #12003 from yersan/bugs/WFLY-11592

[WFLY-11592] Missing microprofile-metrics-smallrye subsystem in some configuration files

kabir committed:
Merge pull request #12004 from spyrkob/WFLY-11599

[WFLY-11599] Upgrade jboss-ejb-client from 4.0.12.Final to 4.0.13.Final

kabir committed:
Merge pull request #12002 from jstourac/jaspiAttrDesc

[WFLY-11593] Typo and ambiguity in 'integrated-jaspi' attribute of 'a…

kabir committed:
Merge pull request #12001 from yersan/tasks/WFLY-11591

[WFLY-11591] Remove org.glassfish:javax.json and javax.json:javax.jso…

kabir committed:
Merge pull request #11989 from spyrkob/WFLY-11576

[WFLY-11576] Upgrade common-digester to 1.8.1

kabir committed:
Merge pull request #11986 from jbliznak/wfly11573-remove-servlet-3.1

[WFLY-11573] Remove unused Servlet API 3.1 dependency

yersan committed:
[WFLY-11602] Add missing configurations to legacy builds
- Missing org.wildfly.extension.microprofile.metrics-smallrye exclusion
- Missing wildfly.undertow.statistics-enabled property to configure the statistics in undertow subsystem in some configurations
- Missing wildfly.messaging-activemq.statistics-enabled property to configure statistics in messaging-activemq subsystem for messaging-activemq-colocated example file

jmesnil committed:
[WFLY-11603] Fix intermittent exception in MP Config SubsystemDeploymentProcessor

Use DeploymentPhaseContext#addDeploymentDependency to inject the
ConfigProviderResolver in the SubsystemDeploymentProcessor
to ensure the service is properly installed and started.

JIRA: https://issues.jboss.org/browse/WFLY-11603

ehsavoie committed:
[WFLY-11142]: Regression in SOAP over JMS when WF13 and WF14 are communicating.

* Adding missing dependency for the wildfly-client-properties artifact

Jira: https://issues.jboss.org/browse/WFLY-11142

hpehl commented:
@spriadka Any chance that these issues will have test dev done, before WF16?

kabir committed:
Merge pull request #11913 from yersan/subtask-5966/WFLY-11467

[WFLY-11467] Validate requirement for modules previously exported by javax.ejb.api on org.jboss.iiop-client

kabir committed:
Merge pull request #12012 from jmesnil/WFLY-11603_NPE_SubsystemDeploymentProcessor

[WFLY-11603] Fix intermittent exception in MP Config SubsystemDeploymentProcessor

kabir committed:
Merge pull request #12010 from jmesnil/WFLY-11588_registerCapabilities

[WFLY-11588] eliminates overrides of SimpleResourceDefinition.registerCapabilities

pferraro committed:
WFLY-11612 Not possible to configure more than 1 passivation-store per deployment with same backing cache-container/cache.

tadamski committed:
[WFLY-11614] IIOP: GIOP's default SSL component constructor fix

bstansberry committed:
Merge pull request #12011 from yersan/tasks/WFLY-11602

[WFLY-11602] Add missing configurations to legacy builds

bstansberry committed:
Merge pull request #12015 from pferraro/WFLY-11612

WFLY-11612 Not possible to configure more than 1 passivation-store per…

darranl committed:
[WFLY-7186] Minor documentation corrections.

bstansberry committed:
Merge pull request #11914 from darranl/WFLY-7186

[WFLY-7186] EE Security API Integration with WildFly Elytron

bstansberry committed:
Merge pull request #11363 from soul2zimate/WFLY-10542

 [WFLY-10542] log thread stack trace which is processing timer task in debug level to diagnose timer overlap

bstansberry committed:
Merge pull request #11704 from tmiyargi/JBEAP-15026

WFLY-11094 Elytron does not do RunAs identity remote propagation

bstansberry committed:
Merge pull request #11965 from pferraro/ejb

WFLY-11530, WFLY-11534,  WFLY-11545 Distributable @Stateful EJB integration fixes and cleanup

bstansberry committed:
Merge pull request #11996 from bstansberry/WFLY-11587

[WFLY-11587] Fix Weld integration with legacy security

bstansberry committed:
Merge pull request #12016 from tadamski/WFLY-11408

[WFLY-11614] IIOP: GIOP's default SSL component constructor fix

manovotn committed:
WFLY-11616 Add privileged block to Principal retrieval when SM is present.

rhusar committed:
WFLY-11449 Upgrade JGroups to 4.0.16.Final

rhusar committed:
Revert "WFLY-11169 Default ASYM_ENCRYPT asym_keylength is considered breakable"

This reverts commit b4de29209581113f07c8a78ff8c4683c2a2fb9a8.

bstansberry committed:
Merge pull request #12021 from rhusar/WFLY-11449

WFLY-11449 Upgrade JGroups to 4.0.16.Final

bstansberry committed:
Merge pull request #12013 from ehsavoie/WFLY-11142

[WFLY-11142]: Regression in SOAP over JMS when WF13 and WF14 are communicating.

bstansberry committed:
Merge pull request #11999 from jbaesner/WFLY-11590

WFLY-11590 Cancelled timer not removed from internal cache when using…

pferraro committed:
WFLY-11626 Reduce unnecessary per-request array allocation/copies when parsing jsessionid

pferraro committed:
Add system property override for session/route delimiter.

jmesnil committed:
Merge pull request #11797 from ehsavoie/WFLY-10493

[WFLY-10493]: Support JMS Java EE resources definition for remote Artemis-based broker.

jmesnil committed:
[WFLY-11637] Upgrade WildFly Core 8.0.0.Beta3

JIRA: https://issues.jboss.org/browse/WFLY-11637

manovotn committed:
WFLY-11636 WeldClassFileInfo should account for meta-annotations.

kabir committed:
Merge pull request #11992 from sgilda/fix-source-blocks

Set appropriate language for source blocks and set nowrap option

ehsavoie committed:
[WFLY-10212]: Deploying of jms bridge requires a lot of mandatory parameters.

* Defining default values for:
  - quality-of-service
  - failure-retry-interval
  - max-retries
  - max-batch-size
  - max-batch-time

Jira: https://issues.jboss.org/browse/WFLY-10212

bstansberry committed:
Merge pull request #12025 from jmesnil/WFLY-11637_upgrade_wildfly-core_8.0.0.Beta3

[WFLY-11637] Upgrade WildFly Core 8.0.0.Beta3

fl4via committed:
[WFLY-11615] Upgrade jboss-ejb-client-legacy to 3.0.3.Final

yersan committed:
[WFLY-11647] Replace hardcoded groupid for wildfly-client-properties in Web Services Tests Integration Subsystem

kabir committed:
Merge pull request #11949 from jmesnil/WFLY-11529_expose_wildfly_metrics

[WFLY-11529] Expose WildFly metrics in Prometheus HTTP endpoint

Commit:
[WFLY-11594] Upgrade MicroProfile Metrics to 1.1.3

marekkopecky committed:
[WFLY-11594] Update Application class name in MP Metrics test

bstansberry committed:
Merge pull request #12033 from yersan/tasks/WFLY-11647

[WFLY-11647] Replace hardcoded groupid for wildfly-client-properties …

marekkopecky committed:
[WFLY-11499] MicroProfileMetrics tests fails with security manager

ehsavoie committed:
[WFLY-11668]: Core queue creation is missing a routing type attribute.

* Adding a routing-type attribute to allow for core-queue creation.

Jira: https://issues.jboss.org/browse/WFLY-11668

bstansberry committed:
@Ignore intermittently failing EJBClientXidTransactionTestCase.testServerSuspension()

kabir committed:
Merge pull request #12039 from bstansberry/ignore-WFLY-11670

@Ignore intermittently failing EJBClientXidTransactionTestCase.testSe…

kabir committed:
Merge pull request #12038 from ehsavoie/WFLY-11668

[WFLY-11668]: Core queue creation is missing a routing type attribute.

kabir committed:
Merge pull request #12028 from fl4via/WFLY-11615

[WFLY-11615] Upgrate jboss-ejb-client-legacy to 3.0.3.Final

tmiyargi committed:
[WFLY-11634] RunAsWithElytronEJBContextPropagationTestCase fails with node0 set

bstansberry committed:
Merge pull request #11895 from scottmarlow/WFLY-11436_nonxa

WFLY-11436 For WildFly 16, also remove NON_XA from WildFly high availability configuration

bstansberry committed:
Merge pull request #11993 from tmiyargi/WFLY-11584

[WFLY-11584] Legacy Web migrate op fails if a connector has scheme ht…

bstansberry committed:
Merge pull request #12019 from manovotn/wfly11616

WFLY-11616 Add privileged block to Principal retrieval when SM is pre…

bstansberry committed:
Merge pull request #12024 from pferraro/master

WFLY-11626 Reduce unnecessary per-request array allocation/copies when parsing jsessionid

bstansberry committed:
Merge pull request #12026 from manovotn/wfly11636

WFLY-11636 WeldClassFileInfo should account for meta-annotations.

bstansberry committed:
Merge pull request #12027 from ehsavoie/WFLY-10212

[WFLY-10212]: Deploying of jms bridge requires a lot of mandatory parameters.

bstansberry committed:
Merge pull request #12040 from tmiyargi/WFLY-11634

[WFLY-11634] RunAsWithElytronEJBContextPropagationTestCase fails with…

ehsavoie committed:
[WFLY-11668-fix]:Issue in WFLY-11668 transformer registration.

* Fixing wrong transformer registration.

Jira: https://issues.jboss.org/browse/WFLY-11668

jmesnil committed:
[WFLY-11677] Upgrade WildFly Core 8.0.0.Beta4

JIRA: https://issues.jboss.org/browse/WFLY-11677

bstansberry committed:
Merge pull request #12043 from jmesnil/WFLY-11677_upgrade_wildfly-core_8.0.0.Beta4

[WFLY-11677] Upgrade WildFly Core 8.0.0.Beta4

bstansberry committed:
Merge pull request #11990 from mchoma/WFLY-11479

[Requires ELY-1466] [WFLY-11479] Tests for HTTP Basic Authentication - Silent Operation

bstansberry committed:
Merge pull request #11997 from gaol/WFLY-11352

[WFLY-11352] WildFly registers multiple distinct drivers for current MySQL driver jar

kabir committed:
Merge pull request #12008 from pferraro/WFLY-11597

WFLY-11597 Allow eager SFSB passivation when enabled via system property

kabir committed:
Merge pull request #12046 from ehsavoie/WFLY-11668-fix

[WFLY-11668-fix]:Issue in WFLY-11668 transformer registration.

bstansberry committed:
Merge pull request #12036 from tterem/microprofile-metrics-issues-tests

[WFLY-11594] Upgrade MicroProfile Metrics to 1.1.3 + tests

bstansberry committed:
Merge pull request #11972 from Skyllarr/WFLY-11479

[WFLY-11479] Add documentation note about silent mode for HTTP BASIC auth

hpehl committed:
WFLY-11408: Upgrade HAL to 3.1.2.Final

hpehl committed:
Merge branch 'WFLY-11408' of github.com:hpehl/wildfly into WFLY-11408

hpehl commented:
Will create a new PR with just the relevant commit.
